salt/apache/iaticdfdbackend.conf.include

```diff
@@ -0,0 +1,49 @@
+# vi: ft=apache
+
+
+        DocumentRoot {{ webserverdir }}
+
+        {{ banner }}
+
+        <Location "/">
+                ProxyPass uwsgi://127.0.0.1:{{ uwsgi_port }}/ timeout={{ uwsgi_harakiri + 15 }}
+        </Location>
+
+        {% if pillar.dev_robots_txt %}
+        Alias /robots.txt "/var/www/html/robots.txt"
+        <Location "/robots.txt">
+                ProxyPass !
+                Order allow,deny
+                Allow from all
+                SetHandler None
+        </Location>
+        {% endif %}
+
+        <Directory {{ webserverdir }}>
+                Require all granted
+        </Directory>
+
+        <Location "/.well-known/acme-challenge">
+                ProxyPass !
+        </Location>
+
+        <Location "/docs">
```

Comment:
Think /docs and /error are specific IATI data store things that aren't needed here? But we can tidy that up later.